src/uu/du/src/du.rs

```diff
@@ -335,6 +336,9 @@ fn du(
 
                             if let Some(inode) = this_stat.inode {
                                 if inodes.contains(&inode) {
```

Comment:
So this `inodes` is the inodes we've seen before? Maybe we can make that name more explicit?

Reply:
Yes. I renamed `inodes` to `seen_inodes`, hope this is a better name.